plugin-api/Examples/hello-world/plugin.html

```diff
@@ -19,13 +19,11 @@
     (function () {
       // Call register function to tell UI where application should be rendered. Callback will be called after
       // plugin is placed to correct spot in UI.
-      Metaflow.register({ slot: 'run-header', container: 'titled-container'}, () => {
+      Metaflow.register({ slot: 'run-header', container: 'titled-container'}, (config, resource) => {
         // Set fixed height for plugin
         Metaflow.setHeight(40);
-        // Subscribe to any update on run data and show run number in plugin.
-        Metaflow.subscribe(['run'], (event) => {
-          document.getElementById('container').innerHTML = `Hello World! Run number is ${event.data.run_number}`;
-        });
+        // Show run id from URL.
```

Comment:
You don't need to subscribe to "metadata"?

Reply:
Removed subscribe due not being needed in this example anymore